update_v1.05.sql

```diff
@@ -0,0 +1,166 @@
+UPDATE `admin_controls_methods` SET `id` = 2477,`order` = 62 WHERE `admin_controls_methods`.`id` = 2477;
+UPDATE `admin_controls_methods` SET `id` = 2478,`order` = 40 WHERE `admin_controls_methods`.`id` = 2478;
+UPDATE `admin_controls_methods` SET `id` = 2479,`order` = 41 WHERE `admin_controls_methods`.`id` = 2479;
+UPDATE `admin_controls_methods` SET `id` = 2480,`order` = 42 WHERE `admin_controls_methods`.`id` = 2480;
+UPDATE `admin_controls_methods` SET `id` = 2481,`order` = 65 WHERE `admin_controls_methods`.`id` = 2481;
+UPDATE `admin_controls_methods` SET `id` = 2482,`order` = 46 WHERE `admin_controls_methods`.`id` = 2482;
+UPDATE `admin_controls_methods` SET `id` = 2483,`order` = 66 WHERE `admin_controls_methods`.`id` = 2483;
+UPDATE `admin_controls_methods` SET `id` = 2484,`order` = 68 WHERE `admin_controls_methods`.`id` = 2484;
+UPDATE `admin_controls_methods` SET `id` = 2485,`order` = 69 WHERE `admin_controls_methods`.`id` = 2485;
+UPDATE `admin_controls_methods` SET `id` = 2486,`order` = 70 WHERE `admin_controls_methods`.`id` = 2486;
+UPDATE `admin_controls_methods` SET `id` = 2487,`order` = 71 WHERE `admin_controls_methods`.`id` = 2487;
+UPDATE `admin_controls_methods` SET `id` = 2488,`order` = 15 WHERE `admin_controls_methods`.`id` = 2488;
+UPDATE `admin_controls_methods` SET `id` = 2489,`order` = 24 WHERE `admin_controls_methods`.`id` = 2489;
+UPDATE `admin_controls_methods` SET `id` = 2492,`order` = 48 WHERE `admin_controls_methods`.`id` = 2492;
+UPDATE `admin_controls_methods` SET `id` = 2493,`order` = 50 WHERE `admin_controls_methods`.`id` = 2493;
+UPDATE `admin_controls_methods` SET `id` = 2494,`order` = 18 WHERE `admin_controls_methods`.`id` = 2494;
+UPDATE `admin_controls_methods` SET `id` = 2495,`order` = 49 WHERE `admin_controls_methods`.`id` = 2495;
+UPDATE `admin_controls_methods` SET `id` = 2496,`order` = 51 WHERE `admin_controls_methods`.`id` = 2496;
+UPDATE `admin_controls_methods` SET `id` = 2497,`order` = 17 WHERE `admin_controls_methods`.`id` = 2497;
+UPDATE `admin_controls_methods` SET `id` = 2498,`order` = 21 WHERE `admin_controls_methods`.`id` = 2498;
+UPDATE `admin_controls_methods` SET `id` = 2499,`order` = 19 WHERE `admin_controls_methods`.`id` = 2499;
+UPDATE `admin_controls_methods` SET `id` = 2500,`order` = 20 WHERE `admin_controls_methods`.`id` = 2500;
+
+UPDATE `app_configuration` SET `btc_currency_id` = '28',`deposit_bitcoin_desc` = '4',`default_fee_schedule_id` = '1',`history_buy_id` = '2',`history_deposit_id` = '4',`history_login_id` = '1',`history_sell_id` = '3',`history_withdraw_id` = '5',`order_type_ask` = '2',`request_awaiting_id` = '4',`request_cancelled_id` = '3',`request_completed_id` = '2',`order_type_bid` = '1',`request_deposit_id` = '2',`request_pending_id` = '1',`request_withdrawal_id` = '1',`transactions_buy_id` = '1',`transactions_sell_id` = '2',`withdraw_fiat_desc` = '1',`withdraw_btc_desc` = '2' WHERE `app_configuration`.`id` = 1;
```

Comment:
What does this line do?  It looks like it's setting several fields related to BTC processing.. But I can't tell what the new values are.